lib/spack/spack/test/compilers/conversion.py

```diff
@@ -6,9 +6,11 @@
 
 from spack.compilers.config import CompilerFactory
 
+pytestmark = [pytest.mark.usefixtures("mock_compilers")]
+
 
 @pytest.fixture()
-def mock_compiler(mock_executable):
+def mock_gcc_yaml(mock_executable):
```

Comment:
Let's continue to call it `mock_compiler`, or if we want to be more specific `mock_legacy_compiler`. The compiler part refers to the deprecated `compiler:` section of the configuration that these tests are about.

Reply:
Switching/switched to `legacy_compiler_yaml` since the change was motivated by my I adding a different `mock_compilers` to `conftest.py`.